src/components/HelpSection.vue

```diff
@@ -242,7 +242,8 @@ export default {
                 font-weight: 600;
 
                 &:hover {
-                    color: #363636;
+                    text-decoration: underline;
+                    text-decoration-color: #363636;
```

Comment:
The first line is what we need, @neeraj-2 . Can you remove the `text-decoration-color`. The underline should be the same as the link itself.

Reply:
Updated!